docs/articles/book/concepts/TheWhy.md

```diff
@@ -0,0 +1,39 @@
+
+And there are the longer-term considerations:
+
+* **Tested code is testable code**. Is our code straight-forward and modular enough to be tested? One sure way to prove this is by testing it.
+* **Thinking about how code will be consumed**. Writing tests forces you to think about calling the code you're writing, which helps you see the surface area and usage of the code. This perspective shift can lead to a better experience for someone consuming your code -- even if that someone is you.
+
+## Why TDD?
+
+TDD -- Test-Driven Development (or sometimes Test-Driven _Design_) -- is the art of writing a test before you write the corresponding piece of production code. Why do that?
+
+* **Timeliness**. Returning to code after we've written it to write tests proves how the program works now. But what about our intent? How can we prove that the program works how we intend it to work? This is where TDD comes in; by writing tests at the time we write the code, we gain additional confidence from testing from our intention all the way through to production code.
+* **Design Drivers**. As we mentioned earlier, some benefits of tests are that they often keep code simpler and more modular, and that they force the developer to test-drive the usage of their code. TDD takes this to the next level -- from early on, you are engaging with your the shape of your code. As you approach the design of your code with TDD, it often becomes easier to spot dependencies as they emerge and make them explicit. By performing these cycles early and often, you gain many opportunities for insight into your code and its usage, and your design will benefit from that.
+
+## Why NUnit?
+
+NUnit is a testing framework for .NET. Like other frameworks -- xUnit, MSTest -- it provides a few core components that compose such a framework:
+
+* Ways to define tests
+* A runner to run the tests (though you can hook into the Visual Studio test runner, the ReSharper test runner, NCrunch, or any other runner of your choice)
+* An assertion library to use during testing (though you can substitute for [FluentAssertions](https://fluentassertions.com/) or the library of your choice)
+
+We believe the concepts we'll discuss here will be able to be applied to any major test framework in most languages. If you couple the concepts in this guide with some documentation in any major test framework, we hope you'll be able to get where you need to be.
+
+So, the question becomes -- who will benefit from NUnit's particular _flavor_ of test framework?
+
+* NUnit benefits in particular from its longevity -- our first NuGet package was published in 2011, 13 years prior to this article being written. In that time, the library has amassed 200+ million downloads and has seen a ton of support from the community. To borrow the Farmers Insurance ad campaign, "We know a thing or two because we've seen a thing or two."
```

Comment:
NUnit dates back to 1998 (I think), but picked up speed around 2002 with NUnit 2.2.  That means actually 25 years of experience :-)